Update index.mdx (#5395)
EnterpriseDB/docs: product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx

### superuser

> Superuser access in BigAnimal is available only where the users are in control of their infrastructure. When using your own cloud account, you can grant the edb_admin role superuser privileges for a cluster. See [Superuser access](/biganimal/latest/getting_started/creating_a_cluster/#superuser-access). If you grant superuser privileges, you must take care to limit the number of connections used by superusers to avoid degraded service or compromising availability, as BigAnimal reserves a few superuser connections for the readiness probe checks if the database is up and able to accept connections, the creation of specific roles required in PostgreSQL instances and some extensions.

Reviewer: Hey! I think we can be a bit more clear in this section. I edited a lot, without knowing much about the context, so feel free to change the parts that are incorrect: 

```suggestion
Superuser access in BigAnimal is only available to users that have complete access to the underlying cloud account ([BigAnimal](/biganimal/latest/planning/deployment_options/#using-biganimals-cloud-account) or [your own cloud account](/biganimal/latest/planning/deployment_options/#using-your-cloud-account)). If you're using your own cloud account, you can assign superuser privileges to the edb_admin role of a cluster. See [Superuser access](/biganimal/latest/getting_started/creating_a_cluster/#superuser-access). 

When granting superuser privileges, ensure you limit the number of connections used by superusers to avoid degrading service or compromising availability. BigAnimal reserves and requires a few superuser connections for the readiness probe, to check if the database is up and able to accept connections, and for the creation of specific roles required in PostgreSQL instances and some extensions. 
```

Author: @gvasquezbargas Can we please not edit the existing content and only add the following part to highlight the reserved superuser connections, thanks!

> When granting superuser privileges, ensure you limit the number of connections used by superusers to avoid degrading service or compromising availability. BigAnimal reserves and requires a few superuser connections for the readiness probe, to check if the database is up and able to accept connections, and for the creation of specific roles required in PostgreSQL instances and some extensions.

Reviewer: ```suggestion
Superuser access in BigAnimal is available only where the users are in control of their infrastructure. When using your own cloud account, you can grant the edb_admin role superuser privileges for a cluster. See [Superuser access](/biganimal/latest/getting_started/creating_a_cluster/#superuser-access). 

When granting superuser privileges, ensure you limit the number of connections used by superusers to avoid degrading service or compromising availability. BigAnimal reserves and requires a few superuser connections for the readiness probe, to check if the database is up and able to accept connections, and for the creation of specific roles required in PostgreSQL instances and some extensions. 
```
